app/views/patron_requests/_request_preferences.html.erb

```diff
@@ -0,0 +1,25 @@
+<% unless f.object.holdable_recallable_items.empty? %>
```

Comment:
I feel like rubocop would have some style suggestions here (but I guess doesn't run on erb files):

Could we flip this condition to an `if` instead of an `unless`?
```suggestion
<% if f.object.holdable_recallable_items.any? %>
```